### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by mikethreels**
### When updating the project the following points have been added.
- [x] Initialized app with create-react-app
- [x] Deleted all unnecessary files or assets.
- [x] Added prop-types using npm.
- [x] Added Redux packages
- [x] Deployed to Heroku.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,2 @@
...
-  </React.StrictMode>,
-  document.getElementById('root')
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// eslint-disable-next-line no-console
+console.log('hello world');
```
Comment (Torres-ssf) [inline src/index.js]: - [x] You did a good job deleting all unnecessary files. However, we need to let `src/index.js` file to remain connected with React.

```JS
import React from 'react';
import ReactDOM from 'react-dom';

ReactDOM.render(
  <React.StrictMode>
    <h1>Hello World</h1>
  </React.StrictMode>,
  document.getElementById('root'),
);

```

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @mikethreels :hand: 
Awesome job on this milestone so far. but we have to make some changes before moving on.

- [x] The live link on your README file it's not working right now, We need to make sure that your app is pushed to Heroku or Netlify.
Screenshot from 2020-11-12 19-36-16

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Comment by mikethreels** [comment]
## Milestone: Basic file structure 

### When updating the project the following points have been added.
- [x] Added live link to the readme file
- [x] Added import from React as requested by the TSE

**Review by lucasmazo23 — APPROVED** [review]
## Project approved 🚀
### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻
## Lucas Mazo

**Pull request merged**